news/6891.feature

```diff
@@ -0,0 +1 @@
+Redact single-part login credentials from URLs
```

Comment:
I would just add a period to the end of this sentence.

Reply:
Done.